db/models/volunteer.js

```diff
@@ -4,7 +4,7 @@ const Sequelize = require('sequelize')
 const db = require('APP/db')
 
 const Volunteer = db.define('volunteers', {
```

Comment:
Are we not asking for email or any kind of contact info? Is this separate from allies/advocates? If so, then what is a volunteer? (Is this only for when they initially contact WWA, then once they're reviewed they get transferred into the ally/advocate table?)